### ISSUE TITLE
Document container targets

### ISSUE THREAD

**Issue by mchack-work** [tillitis/dev-tillitis#47]
In [the build bitstream section](https://dev.tillitis.se/unlocked/build/) in the Dev Handbook we exhort people to use the tkey-builder image for building but don't mention the make targets under `contrib`, like `run-make`. We probably should mention them.

Perhaps we also could change the quickstart things in tillitis-key1 to mention this?

**Comment by dehanj** [comment]
I did some improvements and corrections in #48 

I added some contrib targets, but I would say that we needs to make another tillitis-key1 release in order to make this better - the simplest without building a script is to add a target something like this:

```
﻿﻿build-unlocked:
        podman run --rm --mount type=bind,source="`pwd`/../hw/application_fpga",target=/build -w /build -it \
         $(IMAGE) make clean secret application_fpga.bin
```
(the name of the target we can discuss).

Then I would propose that we add a requirement that one need to have make installed in path. Then only one make command is needed to build the entire firmware with the generated secret, and we no longer need the user to jump around inside a container to fire commands.

It would maybe be even better to add a script that either calls the make target above, and checks that podman, make etc exists in path too. Or simply calls the podman command to remove the need for make? But then we have another script to maintain.

**Comment by mchack-work** [comment]
I agree that it would be very nice to add a script. I'll check out the PR.
